test/unit/default_responses.js

```diff
@@ -111,7 +95,7 @@ exports['pass filter when message is not from gateway'] = function(test) {
 };
 
 exports['do nothing if reported date is not after config start date'] = function(test) {
-    testUtils.restore([transition._isReportedAfterStartDate]);
+    transition._isReportedAfterStartDate.restore();
```

Comment:
For this exact test it's because you're re-stubbing the function just below. For the others though, I'm actually not that sure. Deleting the individual restores breaks. I'll look into it.